### ISSUE TITLE
Default omz theme conflicts with Starship prompt

### ISSUE THREAD

**Issue by kifirkin** [xxh/xxh-plugin-zsh-ohmyzsh#7]
It seems that when using https://github.com/izissise/xxh-plugin-prerun-starship and https://github.com/xxh/xxh-plugin-zsh-ohmyzsh at the same time, setting `ZSH_THEME` here https://github.com/xxh/xxh-plugin-zsh-ohmyzsh/blob/master/pluginrc.zsh#L10-L14 conflicts with starship and overwrites the promt.

Actualy I've just removed that lines and it worked, but I have no idea how to fix this properly...

**For community:**
⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

**Comment by anki-code** [comment]
Good catch. We need to check if ZSH_THEME already exists before setting the default. PR is welcome!
